theme/templates/accounts/profile.html

```diff
@@ -162,8 +162,8 @@ <h4 class="text-muted">
                                     {# Organization #}
                                     <fieldset class="col-sm-6">
                                         <label>Organization *</label>
-                                        <input class="form-control form-required" {% if profile.organization %}value="{{ profile.organization }}"{% endif %} name="organization">
-                                        <small class="text-muted">The name of the organization you work for.</small>
+                                        <input class="form-control form-required tagsinput" {% if profile.organization %}value="{{ profile.organization }}"{% endif %} name="organization">
+                                        <small class="text-muted">The name of the organization(s) you work for. Maximum of two.</small>
```

Comment:
A suggestion for your consideration to change the text to something like ```The name of at most two organizations you work for.``` Looping in @dtarb @horsburgh to get their quick input.

Reply:
"The names of up to two organizations you work for" seems better English to me.  But this opens other problems.  Why is the limit 2.  Why not 1 or 3 or 4.  What if someone does not "work for" an organization, but "attends" or "is studying at", as in the case of a student at a University.  Perhaps we should consider "Your Organization(s)"

Could someone deploy this branch on a staging machine so I can see the broader context of what we are asking for and what the information is that we are collecting on people.

Reply:
@dtarb I'll be happy to throw it up on Park or Playground once the broader context is demonstrable and it's been QA'd internally.

I chose 2 because we need to choose something finite and out of all the users that we have, most have entered nothing for the organization, a minority have entered anything for their organization, and a hyper-minority (like, less than 10) have entered exactly two.

All that being said I will change the help text to "The names of up to two organizations you work for," that sounds better.